MEGA/Main.ino

```diff
@@ -47,15 +47,33 @@ State state();
       int baudRate = 9600;
 
     public:
+      char command;
```

Comment:
Might be best to make this a private variable (under `private:`) considering it'll only be accessed in the Comms class